src/PodiumdAdapter.Web/Infrastructure/ESuiteClient.cs

```diff
@@ -108,6 +108,11 @@ public static void MapEsuiteEndpoints(this IEndpointRouteBuilder builder)
                 var clientSecret = config.GetRequiredValue("ESUITE_CLIENT_SECRET");
                 var token = GetToken(clientId, clientSecret);
                 x.ProxyRequest.Headers.Authorization = new AuthenticationHeaderValue("Bearer", token);
+                var username = x.HttpContext.User.FindFirstValue("user_id");
+                if (!string.IsNullOrWhiteSpace(username))
```

Comment:
maar als hij wel null is. wat dan? lijkt me een error waard

Reply:
[In de authenticatie met een ZGW token is die claim verplicht](https://github.com/ICATT-Menselijk-Digitaal/podiumd-adapter/blob/f5e64fe043dc34fe8565e8d18c4203e5cb7d793c/src/PodiumdAdapter.Web/Infrastructure/AuthExtensions.cs#L32). Als KISS authentiseert met alleen een secret, hebben we die claim niet (en is-ie dus ook niet verplicht).